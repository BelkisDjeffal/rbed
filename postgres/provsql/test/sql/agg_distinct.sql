\set ECHO none
SET search_path TO provsql_test, provsql;

CREATE TABLE agg_result AS
  SELECT city, count(distinct classification)
    FROM personnel
    GROUP BY city ORDER BY city;

SELECT remove_provenance('agg_result');

SELECT * FROM agg_result ORDER BY city;

SELECT city, aggregation_formula(count,'personnel_name') FROM agg_result ORDER BY city;

DROP TABLE agg_result;
