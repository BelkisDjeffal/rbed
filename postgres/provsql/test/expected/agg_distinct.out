\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

   city   | count 
----------+-------
 Berlin   | 1 (*)
 New York | 2 (*)
 Paris    | 3 (*)
(3 rows)

   city   |                     aggregation_formula                     
----------+-------------------------------------------------------------
 Berlin   | count{ (1 * δ((Ellen ⊕ Susan))) }
 New York | count{ (1 * δ(John)) , (1 * δ(Paul)) }
 Paris    | count{ (1 * δ(Nancy)) , (1 * δ(Dave)) , (1 * δ(Magdalen)) }
(3 rows)

